crates/primitives/block/src/header.rs

```diff
@@ -30,12 +30,38 @@ pub struct Header {
     pub event_count: u128,
     /// The version of the Starknet protocol used when creating this block
     pub protocol_version: u8,
-    /// l1 gas price for this block
-    pub l1_gas_price: ResourcePrice,
+    /// Gas prices for this block
+    pub l1_gas_price: GasPrices,
```

Comment:
It will be done in the PR that correctly handle gas price, that will come after we merge this one.
It touch a lot of things accorss the codebase, and can be done separately.